test/integration/apiserver/cel/admission_policy_test.go

```diff
@@ -398,15 +401,38 @@ func createV1ValidatingPolicyAndBinding(client clientset.Interface, convertedRul
 	return nil
 }
 
+// This test shows that policy intercepts all requests for all resources,
+// subresources, verbs, and input versions of policy/binding.
+// The test emulates v1.33 as that was the last version before v1beta1 resource was removed.
+// Remove this test once v1.33 cannot be emulated in v1.37.
+//
+// This test tries to mirror very closely the same test for webhook admission
+// test/integration/apiserver/admissionwebhook/admission_test.go testWebhookAdmission
+func TestPolicyAdmissionV1beta1(t *testing.T) {
+	featuregatetesting.SetFeatureGateEmulationVersionDuringTest(t, utilfeature.DefaultFeatureGate, version.MustParse(vapV1beta1LastEmulatableVersion))
+	testPolicyAdmission(t, true)
```

Comment:
I am generally ok with having separate test for v1beta1 till 1.37 for emulative version. However, isn't the current solution runs duplicated test for v1? Raising this because there was concern raised earlier about the time this kind of test takes.

Reply:
Would you prefer that the duplicate tests are not run for v1 on the previous emulation version? We do technically have coverage for v1 with `TestPolicyAdmission()` on the latest emulation version.

Reply:
Yes if possible. This test takes longer time to run than average. If we could reduce the duplication that will be great. And we could have a clear comment/todo to say that v1beta test coverage will be removed after 1.37. WDT?

Reply:
Updated to test only for v1beta1 and v1 in each test respectively. Comment https://github.com/kubernetes/kubernetes/pull/129564/files#diff-8d6872f37a83db62ebf8d3278fd2ead9097b1cd3194fd0cf933ad15b2e7d1f66R407-R408 indicates that v1beta1 test will be removed in 1.37